man/coredns-proxy.7

```diff
@@ -38,7 +38,7 @@ However, advanced features including load balancing can be utilized with an expa
 .nf
 
 proxy FROM TO\.\.\. {
-    policy random|least_conn|round_robin
+    policy random|least_conn|round_robin|always_first
```

Comment:
How about just `first`?

Reply:
I am ok with 'first'. I will change for it.